addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -351,6 +290,69 @@
     </div>
 </template>
 
+<template id="course_join">
```

Comment:
Can I ask why you moved this whole template?
If it's some kind of preparation work, would be nice to have that in a separate commit maybe (to lower the diff here)?